workers/common/src/main/kotlin/common/OrtMappings.kt

```diff
@@ -191,8 +191,7 @@ fun OrtAdvisorRun.mapToModel(advisorJobId: Long) =
         endTime = endTime.toKotlinInstant(),
         environment = environment.mapToModel(),
         config = config.mapToModel(),
-        advisorRecords = results.advisorResults.mapKeys { it.key.mapToModel() }
-            .mapValues { it.value.map { it.mapToModel() } }
+        advisorRecords = results.advisorResults.map { (k, v) -> k.mapToModel() to v.map { it.mapToModel() } }.toMap()
```

Comment:
Maybe a bit more elegant to do (similar below):
`results.advisorResults.entries.associate { (k, v) -> k.mapToModel() to v.map { it.mapToModel() } }`